src/peergos/server/storage/JdbcTransactionStore.java

```diff
@@ -0,0 +1,103 @@
+package peergos.server.storage;
```

Comment:
Could this get some explicit test coverage (using sqlite)?

Reply:
Done